### PR TITLE
chore: clean up account settings screen

### PR DESCRIPTION
**Pull request by UncleSamtoshi**
- hide phone number
- make upgrade and log in process clearer

### CODE REVIEW

File: e2e/02-login-flow.e2e.spec.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ describe("Login Flow", () => {
 
   it("are we logged in?", async () => {
     // scroll up for small screens
```
Comment (nicolasburtey) [inline e2e/02-login-flow.e2e.spec.ts:92]:
```suggestion
    // scroll down for small screens
```

File: app/graphql/client.tsx [hunk 2]
```diff
@@ -95,12 +95,24 @@ const GaloyClient: React.FC<PropsWithChildren> = ({ children }) => {
         }`,
       )
 
+      const wsLinkAuth = async () => {
```
Comment (nicolasburtey) [inline app/graphql/client.tsx:98]: oh so we need to do a new rollout before activating DeviceAccount?

File: app/graphql/client.tsx [hunk 3]
```diff
@@ -95,12 +95,24 @@ const GaloyClient: React.FC<PropsWithChildren> = ({ children }) => {
...
+        if (token) {
+          return {
+            Authorization: getAuthorizationHeader(token),
+          }
+        } else if (appConfig.isAuthenticatedWithDeviceAccount) {
+          const appCheckToken = await getAppCheckToken()
+          return {
+            Authorization: appCheckToken ? getAuthorizationHeader(appCheckToken) : "",
```
Comment (nicolasburtey) [inline app/graphql/client.tsx:106]:
```suggestion
            Authorization: appCheckToken ? getAuthorizationHeader(appCheckToken) : undefined,
```

shouldn't it be undefined to be consistent with the return below line 109?

File: app/screens/settings-screen/account-screen.tsx [hunk 4]
```diff
@@ -182,14 +187,40 @@ export const AccountScreen = ({ navigation }: Props) => {
     }
   }
 
+  let phoneSettingTitle
+  let phoneSettingAction
```
Comment (nicolasburtey) [inline app/screens/settings-screen/account-screen.tsx:191]: missing types

File: app/graphql/client.tsx (hunk 2 above)
Comment (UncleSamtoshi) [reply app/graphql/client.tsx:98]: This is no longer relevant with required changes to device account authentication

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**